src/Microsoft.AspNetCore.Mvc.Core/Routing/UrlHelper.cs

```diff
@@ -41,10 +41,19 @@ public UrlHelper(ActionContext actionContext)
         /// <inheritdoc />
         public ActionContext ActionContext { get; }
 
+        /// <summary>
+        /// Gets the <see cref="RouteValueDictionary"/>.
+        /// </summary>
         protected RouteValueDictionary AmbientValues => ActionContext.RouteData.Values;
 
+        /// <summary>
+        /// Gets the <see cref="Http.HttpContext"/>
+        /// </summary>
         protected HttpContext HttpContext => ActionContext.HttpContext;
 
+        /// <summary>
+        /// Gets the <see cref="IRouter"/>.
```

Comment:
We should say more here, this is the "top level" router from the current request, which will generally be the route collection.